e2e/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:alpine
+ENV H1_CLI_VERSION="v1.4.0"
+
+# Setup
+RUN apk add curl bats git make gcc musl-dev findutils grep docker
+RUN base=https://github.com/docker/machine/releases/download/v0.16.0 && \
```

Comment:
Any reason not to use the latest docker-machine https://github.com/docker/machine/releases/tag/v0.16.1 ?